dags/utils/slack_operator.py

```diff
@@ -15,74 +18,125 @@ def get_central_time_exec_data(context):
 
 
 def task_fail_slack_alert_critical(context):
-    slack_webhook_token = BaseHook.get_connection(SLACK_CONN_ID).password
```

Comment:
part of the major library version bump

Reply:
ah, so we have `task_fail_slack_alert_critical` and `task_fail_slack_alert`, but we're not using `task_fail_slack_alert_critical` anywhere?

Reply:
my memory @johnclary is that when airflow 2 was set up, the critical dag template was set up as an option/example for future us, if we felt like some dags needed a different type of alert